content/blog/2019/02/2019-02-05-jenkins-new-year-in-china.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Jenkins new year in China"
+tags:
+- core
+- community
+author: linuxsuren
+---
+
+image:/images/2019-spring-festival.jpg[Spring Festival 2019, role=center]
+
+At the time of the Spring Festival. I want to make a summary of some activities in the last year. 
+You might already notice that more and more Chinese contributors emerge in the Jenkins community. 
+We have a GSoC champion who is link:https://github.com/cizezsy[Shenyu Zheng]. 
+He is a great example for other students. With the effort of three skilled engineers, 
+many Jenkins users could learn the edge technologies and useful use cases. 
+They co-orgnaized several Jenkins Meetups in a couple of cities in China.
+
+There are two workshops about Jenkins and Jenkins X in the DevOps International Summit. link:https://github.com/rawlingsj[James Rawlings] gave us a wonderful view of the Jenkins X. Many people start to know this project. The link:https://jenkins-x.io/zh/[Chinese website of jx] would be helpful to those people.
+
+On November 3rd, 2018 the link:/blog/2019/01/09/jenkins-user-conference-china-shenzhen-update/[Jenkins User Conference China(JUCC)] was hosted in Shenzhen. More than 200 attendees gathered at JUCC to share and discuss Jenkins, DevOps, Continuous Delivery, Pipeline, and Agile.
+
+There was a Jenkins workshop to teach users to develop a plugin in October. It was during the Hacktoberfest 2018. So some people got a beautiful T-shirt at this meetup. We'll keep this event in 2019. I hope more users and developers could join us.
+
+Thank you all folks. And other friendly contributors.
+
+image:/images/logos/kongfu/256.png[Chinese KongFu, role=center]
+
+Chinese is our main communication language. A large number of the Jenkins users are not a proficient English speaker. 
+So letting most of Chinese Jenkins users could easily use Jenkins as their CI/CD platform is the final mission of link:https://jenkins.io/sigs/chinese-localization/[Chinese Localization SIG]. 
+You can find three participants on the page. But that's not the link:https://github.com/jenkins-infra/cn.jenkins.io/graphs/contributors[full list]. 
+More exciting thing is that link:https://github.com/alauda[Alauda] giving a big support which as a startup company.
+
+WeChat is the greatest media social channel in China. WeChat has one billion users. 
+Almost everyone has its own account. It must be a perfect place to publish articles and events. 
```

Comment:
"its own account" => using "its" is improper for humans. Probably better as "Almost everyone in China has a WeChat account" or something like this.